Lib/asyncio/streams.py

```diff
@@ -259,6 +265,12 @@ def eof_received(self):
             return False
         return True
 
+    def __del__(self):
+        # Prevent reports about unhandled exceptions.
+        # Better than self._closed._log_traceback = False hack
+        if self._closed.done():
```

Comment:
`if self._closed.done() and not self._closed.cancelled()`

Reply:
Ideally need a test case for the cancelled check.

Reply:
Fixed.
Don you insist on test?
I see no way to make it steady without mocking everything.

Reply:
If it's so hard to hit this then it's OK. I don't like super complex mocked tests anyways.